test/e2e/lib/metrics/stackdriver.go

```diff
@@ -121,7 +121,8 @@ func CheckAssertions(t *testing.T, assertions ...Assertion) {
 				t.Error(err)
 			}
 			return
-		case <-time.After(5 * time.Second):
+		default:
+			time.Sleep(5 * time.Second)
```

Comment:
This change seems unnecessary?

Reply:
The only reason I did that is because I had to reread the original two or three times to convince myself what was going on. I find this clearer, but if you prefer the original, I can switch back.

Reply:
I see, lets go ahead and submit this while the tests are passing since the difference is minimal.